modules/sdk/src/test/resources/ratings.sample.csv

```diff
@@ -0,0 +1,4 @@
+eZNBzdERjxeHdbCRSOyVL,-1
+fKBudHJCIxgGTIZYFSOYwKKvPGIKjt,0
+QyNfuwYquLsVLsjaEixlp,1
+EGHOZIHIgMJsXVRmmVLxBSTEeKJoYwNPYFM,0.5
```

Comment:
still wrong peer ids

Reply:
Can you tell me where to get valid `PeerId`s?

Reply:
I took 4 IDs from https://constellationlabs-dag.s3.us-west-1.amazonaws.com/testnet-seedlist